common/djangoapps/util/cache.py

```diff
@@ -49,7 +50,10 @@ def decorator(view_func):
         @wraps(view_func)
         def wrapper(request, *args, **kwargs):
             """The inner wrapper, which wraps the view function."""
-            if not request.user.is_authenticated():
+            if (
```

Comment:
Could you add a comment before this conditional explaining what's being checked and why?

This makes sense to me but because it's not really features that many of us may be familiar with, I'd like a bit more documentation within the code.